extensions/monday/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "monday.com",
+  "title": "monday.com",
+  "description": "Quickly navigate to your latest monday.com boards and create items",
+  "icon": "command-icon.png",
+  "author": "freak4pc",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Recent Boards",
+      "subtitle": "monday.com",
+      "description": "See your recently interacted with boards, navigate to them, copy their links, or quickly add an item to them",
+      "mode": "view"
+    },
+    {
+      "name": "team",
+      "title": "My Team",
+      "subtitle": "monday.com",
+      "description": "Explore all of your team members",
+      "mode": "view"
+    },
+    {
+      "name": "open",
+      "title": "Go to my monday.com account",
+      "subtitle": "monday.com",
```

Comment:
```suggestion
      "title": "Open Account",
      "subtitle": "monday",
```

Reply:
Open Account might confuse someone to think we're "opening an account for them" (i.e. creating an account for them).

How do you feel about "Navigate to my monday.com account in a browser", or some other more verbose terminology?